docs/application/dotnet/guides/nui/layouting.md

```diff
@@ -0,0 +1,277 @@
+# Layouting
+
+This tutorial describes the NUI Layouting framework, covering the following subjects:
+
+## Layouting Overview
+
+The Layouting framework enables [View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) to be automatically positioned within a parent View that has been assigned a Layout.
+
+The framework offers a common method to layout **Views** with just minimal setup in the application.
+
+Especially, NUI Layouting provides advanced layouting capabilities in a reusable manner. It attempts to separate out the layouting logic from Views into distinct Layouters, so that a View's responsibility is just building and rendering the content.
+
+Layouts are instances of specific Layouts. For example, the first screenshot below shows LinearLayout with Horizontal Layout and Vertical Layout.
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/NUILayoutImage.png">
+</td>
+</tr>
+</table>
```

Comment:
Global comment:

Kindly use MD format to display images. For more infomation, see: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#markdown-style-guide

Reply:
Done.